### DISCUSSION TITLE
percentage of a cluster for certain gene activity

### DISCUSSION THREAD

**Discussion by yxiao832** [GreenleafLab/ArchR#1050] [Questions / Documentation]
Hi, 

To get marker genes in each cluster, I used 

markersGS <- getMarkerFeatures(
    ArchRProj = projHeme2, 
    useMatrix = "GeneScoreMatrix", 
    groupBy = "Clusters",
    bias = c("TSSEnrichment", "log10(nFrags)"),
    testMethod = "wilcoxon"
)

Then I have a SummarizedExperiment object:

class: SummarizedExperiment 
dim: 24919 13 
metadata(2): MatchInfo Params
assays(7): Log2FC Mean ... AUC MeanBGD
rownames(24919): 1 2 ... 24918 24919
rowData names(6): seqnames start ... name idx
colnames(13): C1 C2... C3 C4
colData names(0)

Dose AUC in the assays represent the percent of a cluster is positive for a particular gene?

Thanks!
Yang

**Comment by rcorces** [comment]
No that is not the correct interpretation of the AUC. We do not provide any information about a cell being "positive" for a particular gene because the definition of positive is arbitrary. If you want this information, you'll have to extract it manually with your own arbitrary cutoff.
